repository_service_tuf_worker/repository.py

```diff
@@ -1078,8 +1085,11 @@ def bump_online_roles(self, force: Optional[bool] = False) -> bool:
                 expire (`self._hours_before_expire`)
         """
         logging.debug(f"Configured timeout: {self._timeout}")
-        if self._settings.get_fresh("BOOTSTRAP") is None:
-            logging.info("[automatic_version_bump] No bootstrap, skipping...")
+        bootstrap = self._settings.get_fresh("BOOTSTRAP")
+        if bootstrap is None or "pre-" in bootstrap or "signing-" in bootstrap:
+            logging.info(
+                "[automatic_version_bump] Bootstrap not completed, skipping..."
```

Comment:
"No bootstrap or not completed, skipping"?

Reply:
I think `Bootstrap not completed` could be interpreted by both:
- not started at all
- not signed with threshold of keys

It's good that the user have `GET /api/v1/bootstrap` to understand that bootstrap is in the process of signing: https://github.com/repository-service-tuf/repository-service-tuf-api/blob/277d73b1610a5ec96c4af4d3ccbc8bdbfc69bcf9/repository_service_tuf_api/bootstrap.py#L128
and `GET /api/v1/metadata` to get the exact content of this bootstrap.

In conclusion, I don't we need to change the message to a longer now.